package.json

```diff
@@ -10,30 +10,36 @@
     "default": "./dist/index.modern.js"
   },
   "types": "dist/index.d.ts",
+  "files": [
+    "/dist"
+  ],
```

Comment:
I removed the .npmignore file in favor of the `files` field as we were publishing unnecessary files to npm